internal/cmd/local/k8s/cluster.go

```diff
@@ -110,6 +116,74 @@ func (k *kindCluster) Exists(ctx context.Context) bool {
 	return false
 }
 
+// LoadImages pulls images from Docker Hub, and loads them into the kind cluster.
+// This is a best-effort optimization, which is why it doesn't an error;
+// it's possible that only some images will be loaded.
+// TODO this should probably take a context, and handle cancellation.
+func (k *kindCluster) LoadImages(ctx context.Context, images []string) {
+	err := k.loadImages(ctx, images)
+	pterm.Debug.Printfln("failed to load images: %s", err)
+}
+
+func (k *kindCluster) loadImages(ctx context.Context, images []string) error {
+	// Get a list of Kind nodes.
+	nodes, err := k.p.ListNodes(k.clusterName)
+	if err != nil {
+		return fmt.Errorf("listing nodes: %w", err)
+	}
+
+	// Setup the tar path where the images will be saved.
+	dir, err := fs.TempDir("", "images-tar-")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(dir)
+
+	// Pull all the images via "docker pull", in parallel.
+	var wg sync.WaitGroup
+	wg.Add(len(images))
+	for _, img := range images {
+		pterm.Debug.Printfln("pulling image %s", img)
+
+		go func(img string) {
+			defer wg.Done()
+			out, err := exec.CommandContext(ctx, "docker", "pull", img).CombinedOutput()
```

Comment:
Should we call out to docker or add this functionality to our [internal docker client](https://github.com/airbytehq/abctl/tree/main/internal/cmd/local/docker)?

Same question below on the `docker save` call.

Reply:
Ya, good call, I'll look into that

Reply:
Done